src/main/resources/application.yml

```diff
@@ -1,7 +1,19 @@
 server:
-  port: 8000
+  port: 9000
 hdaf:
   clientregistry:
-    XAuthToken: ${Auth_Token}
+    XAuthToken: 8e983a1719e20913be6c61e9b552e88fc6032c21e1030f1abda986d17326b2b0
     clientId: 6
-    url: ${Client_Registry_Url}
+    url: http://localhost:8080/api/1.0
+  userservice:
+    XAuthToken: 8e983a1719e20913be6c61e9b552e88fc6032c21e1030f1abda986d17326b2b0
+    url: http://localhost:9000
+  hipservice:
+    XAuthToken: 8e983a1719e20913be6c61e9b552e88fc6032c21e1030f1abda986d17326b2b0
+  db:
+    host: localhost
```

Comment:
There will be certificates in prod, need to think about how to configure db based on active profile 'dev/prod/staging', I guess.